### PR TITLE
Update git readme to use standard markdown formatting

### PR DESCRIPTION
**Pull request by cromwellryan**

### CODE REVIEW

File: code-style/git/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Git
 ===
```
Comment (catheraaine) [inline code-style/git/README.md]: This is the wrong heading style, see our heading guidelines!

Comment (catheraaine) [reply code-style/git/README.md]: Great add, would you mind changing this heading while you're in the file?

File: code-style/git/README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 
+Apprentices are the best!
```
Comment (catheraaine) [inline code-style/git/README.md]: Great idea, thanks for adding!

File: code-style/git/README.md (hunk 1 above)
Comment (cromwellryan) [reply code-style/git/README.md]: Yuppers! I'll get this

**Comment by cromwellryan** [comment]
Yeah I probably don't need the apprentice comment in here.

File: code-style/git/README.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
-Git
-===
+# Git
```
Comment (catheraaine) [inline code-style/git/README.md]: Cool.

**Pull request merged**